plugins/block-dynamic-connection/src/dynamic_if.ts

```diff
@@ -188,7 +207,7 @@ const DYNAMIC_IF_MIXIN = {
     if (connection.targetConnection && input.name.includes('IF')) {
       const nextIfInput = this.inputList[inputIndex + 2];
       if (!nextIfInput || nextIfInput.name == 'ELSE') {
-        this.insertElseIf(inputIndex + 2);
+        this.insertElseIf(inputIndex + 2, Blockly.utils.idGenerator.genUid());
```

Comment:
This has to be a UUID to deal with reordering, right?

Reply:
It has to be a UUID to make sure it doesn't conflict with any of the existing names. Any inputs created here are temporary and get deleted in `deleteInputs` so it doesn't really matter for ordering.

I could keep the `inputCount` tracker around to solve this as well. But that seemed confusing since it would only be used for differentiation in this specific case, and not keeping track of the input count in general. I felt this was a better reflection of what I'm actually trying to achieve (a temporary input whose name doesn't conflict with any existing ones).